ui/src/test/java/edu/wpi/grip/ui/codegeneration/WatershedGenerationTest.java

```diff
@@ -85,6 +86,7 @@ void generatePipeline() {
     }
   }
 
+  @Ignore("Grip Watershed is broken so it doesn't make sense to fix the generated watershed.")
```

Comment:
FYI watershed is going to get changed soon (outputting contours instead of an image), so you might want to shelve watershed stuff until that gets merged